# Why: expose the bridge helper used after removing
# SceneBuilder::layer_stack().
diff --git a/engine/src/flutter/shell/common/engine_animator_unittests.cc b/engine/src/flutter/shell/common/engine_animator_unittests.cc
index a310cf9775a..84bed0318ec 100644
--- a/engine/src/flutter/shell/common/engine_animator_unittests.cc
+++ b/engine/src/flutter/shell/common/engine_animator_unittests.cc
@@ -8,6 +8,7 @@
 
 #include "flutter/common/constants.h"
 #include "flutter/lib/ui/compositing/scene_builder.h"
+#include "flutter/runtime_spi/scene/spi_scene_builder.h"
 #include "flutter/shell/common/shell_dart.h"
 #include "flutter/shell/common/shell_test.h"
 #include "flutter/testing/fixture_test.h"
